src-relution/mwFormValidators.js

```diff
@@ -29,7 +34,7 @@
         restrict: 'A',
         require: 'ngModel',
         link: function (scope, elm, attr, ngModel) {
-          var regex = /^\+(?:[0-9]){6,14}[0-9]$/;
+          var regex = /^(00|\+)[0-9()\/-]{6,}$/;
```

Comment:
There is a discussion about that regex and what we should accept as possible numbers.